cla-backend-go/v2/signatures/handlers.go

```diff
@@ -1279,6 +1279,67 @@ func Configure(api *operations.EasyclaAPI, claGroupService service.Service, proj
 		}
 		return signatures.NewInvalidateICLAOK().WithXRequestID(reqID)
 	})
+
+	api.SignaturesEclaAutoCreateHandler = signatures.EclaAutoCreateHandlerFunc(func(eacp signatures.EclaAutoCreateParams, u *auth.User) middleware.Responder {
+		reqID := utils.GetRequestID(eacp.XREQUESTID)
+		ctx := context.WithValue(context.Background(), utils.XREQUESTID, reqID) // nolint
+		utils.SetAuthUserProperties(u, eacp.XUSERNAME, eacp.XEMAIL)
+		f := logrus.Fields{
+			"functionName":   "v2.signatures.handlers.SignaturesEclaAutoCreateHandler",
+			utils.XREQUESTID: ctx.Value(utils.XREQUESTID),
+			"claGroupID":     eacp.ClaGroupID,
+			"companyID":      eacp.CompanyID,
+		}
+		log.WithFields(f).Debug("Updating CCLA signature for the auto_create_ecla column...")
+
+		log.WithFields(f).Debug("Getting corporate signature...")
+		approved := true
+		signed := true
+
+		cclaSignature, err := v1SignatureService.GetCorporateSignature(ctx, eacp.ClaGroupID, eacp.CompanyID, &approved, &signed)
+		if err != nil {
+			msg := "unable to get corporate signature"
+			log.WithFields(f).Warn(msg)
+			return signatures.NewEclaAutoCreateBadRequest().WithXRequestID(reqID).WithPayload(
+				utils.ErrorResponseBadRequestWithError(reqID, msg, err))
+		}
+
+		company, err := companyService.GetCompany(ctx, eacp.CompanyID)
+		if err != nil {
+			msg := "unable to get company"
+			log.WithFields(f).Warn(msg)
+			return signatures.NewEclaAutoCreateBadRequest().WithXRequestID(reqID).WithPayload(
+				utils.ErrorResponseBadRequestWithError(reqID, msg, err))
+		}
+		claGroup, err := claGroupService.GetCLAGroupByID(ctx, eacp.ClaGroupID)
+		if err != nil {
+			msg := "unable to get CLA Group"
+			log.WithFields(f).Warn(msg)
+			return signatures.NewEclaAutoCreateBadRequest().WithXRequestID(reqID).WithPayload(
+				utils.ErrorResponseBadRequestWithError(reqID, msg, err))
+		}
+
```

Comment:
Yes, looks good. Please add logic to check for the current user being in the CLA manager list: https://github.com/communitybridge/easycla/blob/main/cla-backend-go/signatures/service.go#L443-L451